Java/libraries/recognizers-text-date-time/src/main/java/com/microsoft/recognizers/text/datetime/english/extractors/EnglishDateTimeExtractorConfiguration.java

```diff
@@ -0,0 +1,126 @@
+
+import java.util.Arrays;
+import java.util.regex.Pattern;
+
+public class EnglishDateTimeExtractorConfiguration extends BaseOptionsConfiguration implements IDateTimeExtractorConfiguration{
+	
+	public static final Pattern PrepositionRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.PrepositionRegex, Pattern.CASE_INSENSITIVE);
+	public static final Pattern NowRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.NowRegex, Pattern.CASE_INSENSITIVE);
+	public static final Pattern SuffixRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.SuffixRegex, Pattern.CASE_INSENSITIVE);
+	public static final Pattern TimeOfDayRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.TimeOfDayRegex, Pattern.CASE_INSENSITIVE);
+	public static final Pattern SpecificTimeOfDayRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.SpecificTimeOfDayRegex, Pattern.CASE_INSENSITIVE);
+	public static final Pattern TimeOfTodayAfterRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.TimeOfTodayAfterRegex, Pattern.CASE_INSENSITIVE);
+	public static final Pattern TimeOfTodayBeforeRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.TimeOfTodayBeforeRegex, Pattern.CASE_INSENSITIVE);
+	public static final Pattern SimpleTimeOfTodayAfterRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.SimpleTimeOfTodayAfterRegex, Pattern.CASE_INSENSITIVE);
+	public static final Pattern SimpleTimeOfTodayBeforeRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.SimpleTimeOfTodayBeforeRegex, Pattern.CASE_INSENSITIVE);
+	public static final Pattern TheEndOfRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.TheEndOfRegex, Pattern.CASE_INSENSITIVE);
+	public static final Pattern UnitRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.TimeUnitRegex, Pattern.CASE_INSENSITIVE);
+	public static final Pattern ConnectorRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.ConnectorRegex, Pattern.CASE_INSENSITIVE);
+	public static final Pattern NumberAsTimeRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.NumberAsTimeRegex, Pattern.CASE_INSENSITIVE);
+	public static final Pattern DateNumberConnectorRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.DateNumberConnectorRegex, Pattern.CASE_INSENSITIVE);
+	public IExtractor integerExtractor;
+	public IDateTimeExtractor datePointExtractor;
+	public IDateTimeExtractor timePointExtractor;
+	public IDateTimeExtractor durationExtractor;
+	public IDateTimeUtilityConfiguration utilityConfiguration;
+	
+	public EnglishDateTimeExtractorConfiguration(DateTimeOptions options){
+		super(options);
+		integerExtractor = IntegerExtractor.getInstance();
+		datePointExtractor = new BaseDateExtractor(new EnglishDateExtractorConfiguration());
+		timePointExtractor = new BaseTimeExtractor(new EnglishTimeExtractorConfiguration());
+		durationExtractor = new BaseDurationExtractor(new EnglishDurationExtractorConfiguration(options));
+		utilityConfiguration = new EnglishDatetimeUtilityConfiguration();
+	}
+	public EnglishDateTimeExtractorConfiguration(){
```

Comment:
Nit: Code standard. Indentation (whitespace), blank lines between method declarations, whitespace between ) and {, etc.